src/strongbox/ui/jfx.clj

```diff
@@ -521,6 +521,29 @@
                ".table-view#search-addons .updated-column"
                {:-fx-alignment "center"}
 
+               ".tag-button-column-row"
```

Comment:
add note

Reply:
double check these rules actually made a difference.

if not, remove them and remove the styling